docs/admin/install.rst

```diff
@@ -146,33 +150,34 @@ Pango and Cairo
 
 .. versionchanged:: 3.7
 
-Weblate uses Pango and Cairo for rendering bitmap widgets (see
-:ref:`promotion`) and rendering checks (see :ref:`fonts`). To properly install
-Python bindings for those you need to install system libraries first - you need
-both Cairo and Pango, which in turn need GLib. All those should be installed
-with development files and GObject introspection data.
+Weblate uses Pango and Cairo to render bitmap widgets (see
+:ref:`promotion`) and checks (see :ref:`fonts`).
+To properly install Python bindings for those you need to install its system
+libraries first. You need both Cairo and Pango, which in turn need GLib.
+All those should be installed with corresponding development files
+and GObject introspection data.
 
 .. _verify:
 
 Verifying release signatures
 ----------------------------
 
-Weblate release are cryptographically signed by the releasing developer.
-Currently this is Michal Čihař. Fingerprint of his PGP key is:
+Weblate releases are cryptographically signed by the release developer.
+Currently this is Michal Čihař. The fingerprint of his PGP key is:
 
 .. code-block:: console
 
     63CB 1DF1 EF12 CF2A C0EE 5A32 9C27 B313 42B7 511D
 
-and you can get more identification information from <https://keybase.io/nijel>.
+and you can get more identification info from <https://keybase.io/nijel>.
 
-You should verify that the signature matches the archive you have downloaded.
-This way you can be sure that you are using the same code that was released.
+Verify that the signature matches the archive you have downloaded.
+This way you are sure you are using the same code that was released.
 You should also verify the date of the signature to make sure that you
 downloaded the latest version.
 
-Each archive is accompanied with ``.asc`` files which contain the PGP signature
```

Comment:
Fairly sure this is an error, since "files" doesn't correspond to "signature (singular) for it"